hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -211,29 +239,20 @@ public OzoneFSOutputStream createFile(String key, short replication,
       boolean overWrite, boolean recursive) throws IOException {
     incrementCounter(Statistic.OBJECTS_CREATED, 1);
     try {
-      OzoneOutputStream ozoneOutputStream = null;
-      ReplicationConfig replConfig = this.replicationConfig;
-      // Since the bucket has the right default replication, we are using it.
-      if (bucket.getReplicationConfig() != null) {
-        replConfig = bucket.getReplicationConfig();
-      }
-      if (replication == ReplicationFactor.ONE.getValue()
-          || replication == ReplicationFactor.THREE.getValue()) {
-
-        ReplicationConfig customReplicationConfig =
-            ReplicationConfig.adjustReplication(
-                replConfig, replication, config);
-        ozoneOutputStream =
-            bucket.createFile(key, 0, customReplicationConfig, overWrite,
-                recursive);
-      } else {
-        ozoneOutputStream =
-            bucket.createFile(key, 0, replConfig, overWrite, recursive);
-      }
+      ReplicationConfig clientConfiguredReplConfig =
+          this.clientConfiguredReplicationConfig;
+      ReplicationConfig bucketReplConfig =
+          getReplicationConfigWithRefreshCheck();
+
+      OzoneOutputStream ozoneOutputStream = bucket.createFile(key, 0,
+          OzoneClientUtils.resolveClientSideReplicationConfig(replication,
+              clientConfiguredReplConfig, bucketReplConfig, config), overWrite,
+          recursive);
+
       return new OzoneFSOutputStream(ozoneOutputStream.getOutputStream());
     } catch (OMException ex) {
-      if (ex.getResult() == OMException.ResultCodes.FILE_ALREADY_EXISTS
-          || ex.getResult() == OMException.ResultCodes.NOT_A_FILE) {
+      if (ex.getResult() == OMException.ResultCodes.FILE_ALREADY_EXISTS || ex
```

Comment:
This seems to be an accidental reformat, can we revert this please?

Reply:
Fixed it. Yeah unnecessary format.